db/migrate/20240722152432_create_customers_items_join_table.rb

```diff
@@ -0,0 +1,8 @@
+class CreateCustomersItemsJoinTable < ActiveRecord::Migration[7.0]
+  def change
+    create_join_table :customers, :items do |t|
+      t.index :customer_id
+      t.index :item_id
+    end
```

Comment:
Yeah, this will work to join the tables and create those references. You will want to review the rails generator command that helps you with this `rails g migration CreateCustomerItems customer:references item:references` as that will set it all up for you too.